examples/census-hv-dask.ipynb

```diff
@@ -133,7 +145,9 @@
    "source": [
     "shape_path = 'data/cb_2015_us_cd114_5m/cb_2015_us_cd114_5m.shp'\n",
     "districts = gv.Shape.from_shapefile(shape_path, crs=crs.PlateCarree())\n",
-    "districts = gv.operation.project_shape(districts)"
+    "districts = gv.operation.project_shape(districts)\n",
+    "districts = hv.Polygons([np.column_stack([xs, ys]) for dist in districts\n",
+    "                     for xs, ys in zip(*gv.util.geom_to_array(dist.data))])"
```

Comment:
That's a bit mysterious.  Why hv.Polygons to replace gv.Shape?

Reply:
True, mysterious but a lot more efficient. That said I can actually make an optimization in geoviews, ensuring that we only take the hit on initialization. Previously it was spending about 1 second on each zoom event to unpack the shape into an array of arrays.

Reply:
Doing that in geoviews does sound like useful; this will *definitely* mystify people and they won't know why they are doing it or why performance is bad if they don't.